src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  @extend %font-default;
```

Comment:
I think you could add default font styles to body element instead of adding it separatelly